### PR TITLE
*: support insecure registries

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@mrunalp @sameo PTAL

File: cmd/crio/config.go [hunk 1]
```diff
@@ -95,6 +95,10 @@ pause_command = "{{ .PauseCommand }}"
 # unspecified so that the default system-wide policy will be used.
 signature_policy = "{{ .SignaturePolicyPath }}"
 
+# insecure_registries is used skip TLS verification when pulling images.
```
Comment (TomSweeneyRedHat) [inline cmd/crio/config.go]: micro nit, "used skip" to "used to skip"

**Comment by mrunalp** [comment]
The fedora failure is a flake. LGTM.

**Comment by runcom** [comment]
ping @sameo

**Comment by sameo** [comment]
LGTM

**Pull request merged**